lib/modules/actions.js

```diff
@@ -8,6 +8,8 @@ export const SET_SHELL = 'PLATFORM__SET_SHELL';
 export const REDIRECT = 'PLATFORM__REDIRECT';
 export const REROUTE_PAGE = 'PLATFORM__REROUTE_PAGE'; // re-run the handlers for the page
 
+export const NO_REFERRER  = 'NO_REFERRER';
```

Comment:
As commented below - the field isn't empty (maybe we should define what 'empty' means) it is an empty string. The point is to be able to distinguish this referrer from the referrer added on a single page view change. 

Maybe then the simplest change that doesn't require changing platform would be to make sure the correct referrer is built and passed in when dispatching a navigateToUrl action. But that would require everyone who calls it to know they should do that. So that doesn't seem great.